File: src/if.rb
print "a? "
a=gets.to_i

if a >= 0
  puts "Positive"
else
  puts "Negative"
  puts "minus"
end

if a > 0
  puts "Positive"
elsif a == 0
  puts "Zero"
else
  puts "Negative"
end

print "name?"
name = gets.chomp
if name == "KSU"
  puts "Hello"
else
  puts "Hi!"
end

q=5
if q==3
  puts "Three"
end
puts "Three" if q==3

puts "Not Three" if q!=3
puts "Not Three" unless q==3

File: src/If.java
import java.util.*;

class If {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("a? ");
        int a=sc.nextInt();
        if(a > 0) {
            System.out.println("Positive");
        } else if(a == 0) {
            System.out.println("Zero");
        } else {
            System.out.println("Negative");
        }
/*
        boolean boo;
        boo=false;
        if(boo=false) {
            System.out.println("true");
        }
        */
       int q=3;
       if(q==3) {
        System.out.println("three");
       }

       String s;
       int a=5;
       /*
       if(a%2 == 0) {
        s = "Even";
       }else{
        s = "Odd";
       }
        */
       s= a%2==0 ? "Even" : "Odd";
       System.out.println(s);
   }
}